### ISSUE TITLE
hint text: increase contrast or remove it as a separate element

### ISSUE THREAD

**Issue by sulthan-ahmed** [alphagov/govuk_elements#250]
Problem: Registered traveller programme, benjystanton (from DVLA), alextorrance_gds have found in their user research that grey is causing some issues from an accessibility perspective.  It is difficult for those with visual impairment to read it.

Alex quite rightly points out that in the examples of elements because think they have to use it everywhere

Solution: 
1. Make hint text black but why then do we need a css class for it?
2. Remove it but that will have an impact to so many gov forms especially for hof based forms like ours?

**Comment by gemmaleigh** [comment]
I think the simplest solution would be to override `color: $secondary-text-colour` set for `.hint-text`.
You need a class for it, as it sets bottom padding - between the hint text and the label.

It would be great if evidence of this could be passed on to @timpaul, who is prioritising work for existing patterns that we need to revise.

**Comment by sulthan-ahmed** [comment]
font-colour not compliant by WCAG standards

![pasted image at 2016_06_09 10_08 am](https://cloud.githubusercontent.com/assets/12494656/15924875/532e8cdc-2e2c-11e6-81c5-fd3bf883252f.png)

**Comment by gemmaleigh** [comment]
That's right, it passes contrast ratio requirements for large text - 18px or above. 
Hint text is the same size as body text, at 19px.

My suggestion above was to override this colour (so use a darker colour, e.g. $text-colour) if the existing .hint-text colour $secondary-text-colour is too light).

**Comment by edwardhorsford** [comment]
@sulthan-ahmed We aim for AA, not AAA. Higher contrast is generally better though.

I'd be cautious about just changing the colour - some services may be relying on people being able to perceive the difference between body and hint text. Would be good to do some research on how hint texts is currently used.

**Comment by sulthan-ahmed** [comment]
@gemmaleigh yes we've done this. @edwardhorsford I agree it needs more thought and concrete data.  However, it does seem that the folks I mentioned above have got some user research that can aid this discussion and ultimately come up with a decision.

**Comment by accessiblewebuk** [comment]
@sulthan-ahmed  as @edwardhorsford says, we require minimum of level AA, but don't currently require above that. It is always difficult to get good colour contrast with background at the same time as good contrast between two types of text with different meaning (but the same background colour for both). This is always a challenge for links if they aren't distinguished in other ways. 

@gemmaleigh - the requirement for large text applies to 18pt or above, not 18px. Recent discussions have indicated that a pt generally equates to 1 and a third px, so large text either needs to be minimum 24px, or minimum 19px if it is bold.

**Comment by sulthan-ahmed** [comment]
thanks @accessiblewebuk @edwardhorsford I understand the minimum level scheme of AA. I hate writing messages rather than speaking in person because they may not come out as the most polite. With the greatest respect to you both, should we look to help our users rather than concentrating on meeting a minimum level?  Should we not look to improve beyond AA? There is evidence from the Home office, DVLA and GDS to say our users have had problems with grey. 

However, I think @edwardhorsford does make a good point about the impact this will have.  I also agree that it would be good to have concrete evidence behind this

**Comment by accessiblewebuk** [comment]
@sulthan-ahmed It is always good to go beyond the guidelines where possible, but we prioritise meeting the minimum across all the success criteria, and making levels of contrast higher could create problems for some users so more research is needed in that area (we know there are users who require lower than normal contrast and there are issues with changing the default settings in Firefox for example, and for some dyslexic users very high contrast can cause problems).

**Comment by sulthan-ahmed** [comment]
fair points @accessiblewebuk 👍

**Comment by nickcolley** [comment]
Following the launch of the [GOV.UK Design System](https://design-system.service.gov.uk/), GOV.UK Elements will now only get major bug fixes and security patches.

Since this has not had any activity for a long time I'm going to close this out since the current colour meets the WCAG Level AA contrast requirements.

If you can get the original research or find this is still a problem please raise this on the GOV.UK Design System backlog so we can look into it: https://github.com/alphagov/govuk-design-system-backlog/issues/51

Thanks for raising :)
